index.js

```diff
@@ -175,9 +202,15 @@ MinkeLite.prototype._init_db = function () {
     false :
     fs.existsSync(this.db_path)
 
+  if(this.db_exists && this.config.over_write) {
+    fs.unlinkSync(this.db_path);
```

Comment:
This is a race condition: if the DB is removed between the check of its existence above, and the attempt to unlink here, then the `unlinkSync()` will throw an error. You should wrap the unlink with try-catch.

Use of the `exists()` is often an anti-pattern for this reason, its almost always racy. Its best to attempt to use a resource and handle its existence/non-existence than to check if it exists at some point in time, and assume that state hasn't changed at some later time.